templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/versions_of_statement.html.twig

```diff
@@ -26,9 +26,11 @@
 
         {% if draftStatementVersions is defined and  draftStatementVersions|length > 0 %}
 
-            {% for index, statement in draftStatementVersions if index < lastIndex %}
-                <h2>{{ "date"|trans }}: {{ statement.versionDate|dplanDate('d.m.Y | H:i') }}</h2>
-                {{ statement.text|wysiwyg }}
+            {% for index, statement in draftStatementVersions%}
+                {% if index < lastIndex  %}
+                    <h2>{{ "date"|trans }}: {{ statement.versionDate|dplanDate('d.m.Y | H:i') }}</h2>
+                    {{ statement.text|wysiwyg }}
+                {% endif %}
```

Comment:
why is this change neccesssary? 
The Doc tells me that it should work the way it was before https://twig.symfony.com/doc/2.x/tags/for.html#adding-a-condition
Could it be, that setting some brackets help to solve the Bug?

Reply:
It gives error with this format

Reply:
We use Twig 3

Reply:
:bulb:  :-/

Reply:
then, what @mussbach says (filter)

Reply:
done in [5f79a5f](https://github.com/demos-europe/demosplan-core/pull/2152/commits/5f79a5f388775759f295fe310d31312f4b32d7bd)